ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -483,10 +491,27 @@ static int logical_immediate_p (uint64_t value, uint32_t reg_size)
 ||	}
 |.endmacro
 
+// Safe memory load/store with an unsigned 64-bit offset.
+|.macro SAFE_MEM_ACC_WITH_64_UOFFSET, ldr_str_ins, op, base_reg, offset, tmp_reg
+||	if (((uintptr_t)(offset)) > LDR_STR_PIMM64) {
+|		LOAD_64BIT_VAL tmp_reg, offset
+|		ldr_str_ins op, [base_reg, tmp_reg]
+||	} else {
+|		ldr_str_ins op, [base_reg, #(offset)]
+||	}
+|.endmacro
+
 |.macro LOAD_TSRM_CACHE, reg
+||#ifdef __APPLE__
+|	.long 0xd53bd071 // TODO: hard-coded: mrs TMP3, tpidrro_el0
+|	and TMP3, TMP3, #0xfffffffffffffff8
+|	SAFE_MEM_ACC_WITH_64_UOFFSET ldr, TMP3, TMP3, (((TLVDescriptor*)tsrm_ls_cache_tcb_offset)->key << 3), TMP1
```

Comment:
Slow path is avoided with the premise that tlv `_tsrm_ls_cache` is guaranteed to be accessed before the JIT-ed.

We have one concern that it would be a bit difficult for debugging if the premise is not satisfied with the software evolution in the future.

@dstogov Do you think we should add one check here?
How to do:
1. add `cbz TMP3, ->trap_label` here
2. define one global label named `trap_label` in one new defined stub function only for macOS case. In this label, `brk #0` is the only instruction.

Impact:
1. Performance: one `cbz` instruction would be executed each time, but it would not impose much overhead with the help of branch prediction.
2. Code size: one `cbz` instruction would be produced for each invocation of macro `LOAD_TSRM_CACHE`.

Pros: In the future, if the premise is missing, it helps to debug.

Reply:
I think, everything fine now. There is no a big difference between debugging ``brk #0`` trap or crash right after ``mrs TMP3, tpidrro_el0``. I prefer not to add ``cbz``.

Reply:
I think the `cbz` is to check the value in `TMP3` after the first load at line 508.
It's possible that the crash site is far away  `mrs`.

Reply:
Recall that the value in `TMP3` after line 508 corresponds to the `x17` after line 5 in https://gist.github.com/shqking/329d7712c26bad49786ab0a544a4af43#file-tlv_get_addr-s-L6